torch/csrc/api/include/torch/nn/options/loss.h

```diff
@@ -153,5 +153,20 @@ struct TORCH_API TripletMarginLossOptions {
   TORCH_ARG(torch::Reduction::Reduction, reduction) = torch::Reduction::Mean;
 };
 
+// ============================================================================
+
+/// Options for a nll-loss functional and module.
+struct TORCH_API NLLLossOptions {
+  /// A manual rescaling weight given to each class. If given, has to be a Tensor
+  /// of size C
+  TORCH_ARG(Tensor, weight) = {};
+  /// Specifies a target value that is ignored
+  /// and does not contribute to the input gradient. When :attr:`size_average` is
+  /// ``True``, the loss is averaged over non-ignored targets.
+  TORCH_ARG(double, ignore_index) = -100;
+  /// Specifies the reduction to apply to the output. Default: Mean
+  TORCH_ARG(Reduction::Reduction, reduction) = Reduction::Mean;
```

Comment:
I think `Reduction` is now under `torch::`,  and we can write
```suggestion
  TORCH_ARG(torch::Reduction::Reduction, reduction) = torch::Reduction::Mean;
```